freqtrade/exchange/exchange.py

```diff
@@ -338,7 +338,7 @@ def get_pair_base_currency(self, pair: str) -> str:
         return self.markets.get(pair, {}).get('base', '')
 
     def market_is_future(self, market: Dict[str, Any]) -> bool:
-        return market.get('swap', False) is True
+        return market.get(self._ft_has["ccxt_futures_name"], False) is True
```

Comment:
ah - i missed this... :+1: 

then please remove the "overwriting" method in `binance.py` and `ftx.py` ...